Cargo.toml

```diff
@@ -39,7 +39,7 @@ os-ext = ["os-poll"]
 net = []
 
 [dependencies]
-log = "0.4.8"
+log = { version = "0.4.16", default-features = false, features = ["kv_unstable"] }
```

Comment:
This is an unstable feature?

Reply:
Yes, but it's been working on for over a year and it's close to table, also see https://github.com/rust-lang/log/issues/436.